schema/atac_schema.md

```diff
@@ -0,0 +1,473 @@
+## scATAC-seq assay types
+
+<i>paired assay</i>: any descendant of <a href="https://www.ebi.ac.uk/ols4/ontologies/efo/classes?obo_id=EFO%3A0010891"><code>"EFO:0010891"</code></a> for <i>scATAC-seq</i> that is also a descendant of <a href="https://www.ebi.ac.uk/ols4/ontologies/efo/classes?obo_id=EFO%3A0008913"><code>"EFO:0008913"</code></a> for <i>single-cell RNA sequencing</i>
+
+<i>unpaired assay</i>: <a href="https://www.ebi.ac.uk/ols4/ontologies/efo/classes?obo_id=EFO%3A0010891"><code>"EFO:0010891"</code></a> for <i>scATAC-seq</i> or its descendants that is not a descendant of <a href="https://www.ebi.ac.uk/ols4/ontologies/efo/classes?obo_id=EFO%3A0008913"><code>"EFO:0008913"</code></a> for <i>single-cell RNA sequencing</i>
+
+## Fragment File Dataset Criteria
+
+A Dataset MUST meet each of the following criteria in order to be eligible for an attached Fragment File:
+* the <code>obs['assay_ontology_term_id']</code> values MUST all be <i>paired assays</i> or MUST all be <i>unpaired assays</i>
+* the <code>obs['is_primary_data']</code> values MUST be all `True`
+* the <code>var['feature_reference']</code> values MUST include one of <code>"NCBITaxon:9606"</code> for <i>Homo sapiens</i> or <code>"NCBITaxon:10090"</code> for <i>Mus musculus</i>, but not both. The value that is present will determine the appropriate Chromosome Table for standards.
+
+If the <code>obs['assay_ontology_term_id']</code> values are all <i>paired assays</i> then a fragment file MAY be attached to the Dataset.
+
+If the <code>obs['assay_ontology_term_id']</code> values are all <i>unpaired assays</i> then a fragment file MUST be attached to the Dataset.
```

Comment:
This requirement will complicate validation in the ingest pipeline. We will have to wait for the fragment file to be present before knowing if the anndata is valid. If more requirements like this are expected in the future then we can pay the engineer cost to simplify change like this for the future.

Reply:
> It would seem easier to simply block publication of the collection if a fragment file is not attached to a dataset?

💯 we need to block in this case. I'm thinking about how the backend will process this when multiple files are required before being a valid dataset.

Reply:
> when multiple files are required before being a valid dataset.

I'm suggesting that the dataset is valid, but the collection is invalid when there's a missing/required fragment file.

* Upload and validate dataset - set collection as invalid due to missing requirement fragment file.
* Upload and validate fragment file - set collection as valid if fragment validation passes. 

Another approach is to always upload+validate the dataset and fragment file together. The two files are tarred or gzipped together.